### PR TITLE
add OptNTPServers

### PR DESCRIPTION
**Pull request by pmazzini**

### CODE REVIEW

File: dhcpv4/option_ntp_servers.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+// String returns a human-readable string.
+func (o *OptNTPServers) String() string {
+	var ntpServers string
+	for idx, ntp := range o.NTPServers {
+		ntpServers += ntp.String()
+		if idx < len(o.NTPServers)-1 {
+			ntpServers += ", "
+		}
+	}
+	return fmt.Sprintf("NTP Servers -> %v", ntpServers)
```
Comment (insomniacslk) [inline dhcpv4/option_ntp_servers.go]: what about just `strings.Join(o.NTPServers, ", ")` ?

Comment (pmazzini) [reply dhcpv4/option_ntp_servers.go]: The problem is that o.NTPServers is not a list of strings.

Comment (insomniacslk) [reply dhcpv4/option_ntp_servers.go]: right - and not even bytes.Join as this should still need explicit casting

**Comment by insomniacslk** [comment]
Can you look at the merge conflict?

**Comment by pmazzini** [comment]
Rebased, I think it should be good now.

**Pull request merged**